fhir-core/src/test/java/com/ibm/fhir/core/test/mains/DateTestMain.java

```diff
@@ -1,96 +0,0 @@
-/*
- * (C) Copyright IBM Corp. 2016,2019
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package com.ibm.fhir.core.test.mains;
-
-import static com.ibm.fhir.core.FHIRUtilities.*;
-import static org.testng.AssertJUnit.*;
-
-import java.util.Arrays;
-import java.util.Date;
-import java.util.GregorianCalendar;
-import java.util.List;
-
-import javax.xml.datatype.DatatypeFactory;
-import javax.xml.datatype.Duration;
-import javax.xml.datatype.XMLGregorianCalendar;
-
-import org.testng.annotations.Test;
-
-import com.ibm.fhir.core.FHIRUtilities;
-
-public class DateTestMain {
-    // TODO remove when we are happy we can permanently delete the FHIRUtilities methods under test
-//    public static void main(String[] args) throws Exception {
-//        List<XMLGregorianCalendar> calendars = Arrays.asList(
-//            parseDateTime("2015-12-31T20:00:00-04:00", false),   // timezone other than GMT
-//            parseDateTime("2015-12-31T20:00:00-05:00", false),   // different timezone other than GMT
-//            parseDateTime("2016-01-01", false),                  // date
-//            parseDateTime("2016-01", false),                     // year month
-//            parseDateTime("2016", false)                         // year
```

Comment:
do we have somewhere else that tests the parsing of values like this into our DateTime object?

Reply:
parseDateTime is defined in the model project now, so I think we should unitest there instead of in the core project.

Reply:
OK, sounds good!

Reply:
@lmsurpre just added test cases for FHIR date, time types

Reply:
done